migrator/bin/ups-migrator

```diff
@@ -0,0 +1,175 @@
+#!/bin/sh
```

Comment:
but is it inteted, to be executed from the checked out source? reading the
instructions, I have the impression the way to go is extracting the -dist
zip file

On Mon, Feb 23, 2015 at 6:42 PM, Douglas Campos <notifications@github.com>
wrote:

> In migrator/bin/ups-migrator
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/502#discussion_r25183911>
> :
>
> > @@ -0,0 +1,175 @@
> > +#!/bin/sh
>
> executable by itself
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/502/files#r25183911>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf